Can't import on MacOS Catalina

Hi! I was trying to run the examples made by ThomasTJdev, and after not being able to run any of the examples, I narrowed the problem to the first line.

> import webgui

It seems this makes clang show a lot of errors that I can't even begin to debug. (Nim beginner here!).

I made [this](https://gist.github.com/agucova/8a433e74671fe576177bd33d8b0a277e) gist with the build log on my machine.

I'm using the latest WebGUI from Nimble (0.4.0), and I'm running Nim 1.2.0 installed by Homebrew in MacOS Catalina 10.15.3. Can try any suggestions if needed.

Running `nim objc minimal.nim` gives the exact same errors:

```
webview.h:1717:23: error: too many arguments to function call, expected 0, have 3
  return objc_msgSend((id)objc_getClass("NSString"),
```

--- a/src/webgui.nim
+++ b/src/webgui.nim
@@ -53,7 +53,7 @@ when defined(linux):
 elif defined(windows):
   {.passc: "-DWEBVIEW_WINAPI=1", passl: "-lole32 -lcomctl32 -loleaut32 -luuid -lgdi32".}
 elif defined(macosx):
-  {.passc: "-DWEBVIEW_COCOA=1 -x objective-c", passl: "-framework Cocoa -framework WebKit".}
+  {.passc: "-DOBJC_OLD_DISPATCH_PROTOTYPES=1 -DWEBVIEW_COCOA=1 -x objective-c", passl: "-framework Cocoa -framework WebKit".}
 
 type
   ExternalInvokeCb* = proc (w: Webview; arg: string)  ## External CallBack Proc

